.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-config.js
\ No newline at end of file
+config.js
+bundle.js
```

Comment:
thanks for adding bundle.js to this Andrew! 👍

Reply:
Remember, this will not work if you have been tracking the bundle.js  changes. In order to make it work you have to go to your terminal in your project directory  and enter : 

git rm --cached client/dist/bundle.js

Without this, bundle.js cannot be ignored.